jax/_src/config.py

```diff
@@ -578,6 +616,33 @@ def _update_x64_thread_local(val):
 
 Config.x64_enabled = Config.jax_enable_x64  # type: ignore
 
+
+def _update_default_device_global(val):
+  lib.jax_jit.global_state().default_device = val
+
+def _update_default_device_thread_local(val):
+  lib.jax_jit.thread_local_state().default_device = val
+
+def _validate_default_device(val):
+  if val is not None and not isinstance(val, xla_client.Device):
+    raise ValueError("jax.default_device must be passed a Device object (e.g. "
+                     f"`jax.devices('cpu')[0]`), got: {repr(val)}")
+
+# TODO(skye): default_device only accepts devices for now. Make it work with
+# platform names as well (e.g. "cpu").
+default_device = config.define_string_or_object_state(
+    name='jax_default_device',
+    default=None,
+    help=(
+        'Set to a Device object (e.g. ``jax.devices("cpu")[0]``) to use as the '
```

Comment:
```suggestion
        'Configure the default device for JAX operations. '
        'Set to a Device object (e.g. ``jax.devices("cpu")[0]``) to use that Device as the '
```

(I thought it might be useful to start with a short description of what this option is, before saying more about how to set it.)